solutions/devsprint-bruno-ramos-5/GitHubApp/Screens/List/ListView.swift

```diff
@@ -12,6 +12,10 @@ struct ListViewConfiguration {
     let listItems: [String]
 }
 
+protocol ListViewProtocol {
```

Comment:
Como você esta utilizando esse protocolo como um delegate, considere explicitar isso no nome do protocolo.
```suggestion
protocol ListViewDelegate {
```

Reply:
Boa noite Professor, arrumei pra delegate. No entanto eu não vejo como um delegate. Vou perguntar durante a aula sobre isso pra tirar minha a dúvida